server/watch.go

```diff
@@ -1,5 +1,54 @@
 package server
 
+import (
+    "context"
+    "log"
+    "time"
+
+    "go.mongodb.org/mongo-driver/mongo"
+    "go.mongodb.org/mongo-driver/mongo/options"
+    "go.mongodb.org/mongo-driver/bson"
+
+    "aux3.xyz/common"
+)
+
 func Watch() {
-    // loop forever watching database for new events
+    // Get the shared MongoDB client
+    client := common.GetMongoClient()
+    ctx := context.Background()
+
+    // Get collection to watch
+    // TODO: setup right database and collection names
+    collection := client.Database("aux3").Collection("events")
+
+    // Set up change stream
+    pipeline := mongo.Pipeline{}
+    opts := options.ChangeStream().SetFullDocument(options.UpdateLookup)
```

Comment:
>Change streams allow applications to access real-time data changes without the prior complexity and risk of manually tailing the [oplog](https://www.mongodb.com/docs/manual/reference/glossary/#std-term-oplog). Applications can use change streams to subscribe to all data changes on a single collection, a database, or an entire deployment, and immediately react to them. Because change streams use the aggregation framework, applications can also filter for specific changes or transform the notifications at will.

https://www.mongodb.com/docs/manual/changeStreams/ (switch language to go in right column)